webtau-reactjs/src/report/OverallSummary.css

```diff
@@ -22,7 +22,7 @@
 }
 
 .overall-http-calls-time,
-.overall-http-calls-coverage {
+.overall-operations-coverage {
```

Comment:
seems like http calls is till valid here? 
need some indication what operations are as this report is also used for UI testing.

Reply:
Yeah, I guess this one probably still make sense as http-calls, will put it back.